Tests/UnitTests/Core/TrackFinding/CombinatorialKalmanFilterTests.cpp

```diff
@@ -225,27 +229,29 @@ BOOST_AUTO_TEST_CASE(ZeroFieldForward) {
 }
 
 // TODO test currently fails w/ a smoothing error
-BOOST_AUTO_TEST_CASE(ZeroFieldBackward, *boost::unit_test::disabled()) {
```

Comment:
Nice. How did you manage to fix the broken test?

Reply:
The test is broken because the starting parameters have not been set properly, i.e. all the three starting parameters are closest to the first track. This has been fixed by adjusting the second(third) starting parameters to make them closer to the second(third) track.

Reply:
Nice catch.